src/main/java/io/split/android/client/impressions/HttpImpressionsSender.java

```diff
@@ -28,40 +29,31 @@ public HttpImpressionsSender(CloseableHttpClient client, String eventsEndpoint,
     }
 
     @Override
-    public void post(List<TestImpressions> impressions) {
+    public boolean post(List<TestImpressions> impressions) {
+        if (impressions == null || impressions.isEmpty()) {
+            return true;
+        }
 
         if (!Utils.isReachable(_eventsEndpoint)) {
             Timber.d("%s is NOT REACHABLE. Avoid trying to send impressions this time.", _eventsEndpoint.getHost());
```

Comment:
how often does this get logged?

Reply:
Only if the bulk impressions endpoint can't be logged?